www-apps/kibana-bin/kibana-bin-7.17.10.ebuild

```diff
@@ -0,0 +1,94 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit systemd
+
+MY_PN="${PN%-bin}"
+MY_P=${MY_PN}-${PV}
+
+DESCRIPTION="Analytics and search dashboard for Elasticsearch"
+HOMEPAGE="https://www.elastic.co/products/kibana"
+SRC_URI="https://artifacts.elastic.co/downloads/${MY_PN}/${MY_P}-linux-x86_64.tar.gz"
+
+# source: LICENSE.txt and NOTICE.txt
+LICENSE="Apache-2.0 Artistic-2 BSD BSD-2 CC-BY-3.0 CC-BY-4.0 Elastic-2.0 icu ISC MIT MPL-2.0 OFL-1.1 openssl public-domain Unlicense WTFPL-2 ZLIB"
+SLOT="0"
+KEYWORDS="-* ~amd64"
+
+RDEPEND="
+	acct-group/kibana
+	acct-user/kibana
+	dev-libs/expat
+	dev-libs/nspr
+	dev-libs/nss
+	>=net-libs/nodejs-16.13.0
+	<net-libs/nodejs-18
```

Comment:
Are you sure about this? This is annoying if true.

Reply:
Last time I checked it was so.